src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -65,6 +65,9 @@ public class Checker extends AbstractAutomaticBean implements MessageDispatcher,
     /** Message to use when an exception occurs and should be printed as a violation. */
     public static final String EXCEPTION_MSG = "general.exception";
 
+    /** One single White space. */
+    public static final String SINGLE_WHITESPACE = " ";
```

Comment:
Why do we need it as a separate field? I'd suggest to use it in place.

Reply:
The string literal "` `" has been used at a total of 4 places in the file. Apparently, we have a check called _MultipleStringLiterals_ that allows no repetitions (by default max allowed = 1).

Reply:
1. Could you make it `private` since it is used within this class only?
2. Please rename it as `WHITESPACE_SEPARATOR`
3. Update comment as follows: `Separator to use in string concatination.`

Reply:
Done..
Actually, I left the field `public static` so that it can be called later when required somewhere else while I work on rest of the files of this issue. I though this way, we could avoid multiple private field declarations for every class :)

Reply:
> later when required somewhere else

This "later" rarely comes in practice. Better and easier to apply restrictions now and loosen them later if needed than the other way around.